platform/view/services/comm/host/rest/websocket/multiplexed_provider.go

```diff
@@ -150,7 +182,7 @@ func newClientConn(conn *websocket.Conn, tracer trace.Tracer, m *Metrics, onClos
 
 func (c *multiplexedClientConn) newClientSubConn(ctx context.Context, src host2.PeerID, info host2.StreamInfo) (*stream, error) {
 	c.mu.Lock()
-	sc := c.newSubConn(NewSubConnId())
+	sc := c.newSubConn(strconv.FormatUint(c.subConnId.Add(1), 10))
```

Comment:
at this point we could extract this to a method `c.newSubConnId() SubConnId` and define SubConnId as int64 instead of string to avoid the conversion. What do you think?

Reply:
yip, we could change this -do you think we should do this as part of this PR?
The main reason why I touched the subConIDs at all is to avoid the package variable and thus keep the scope unique per base connection.